From 44d4060c9d3dcf8d28ba4f4569ba6e0971237cd2 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Caol=C3=A1n=20McNamara?= <caolanm@redhat.com>
Date: Fri, 31 Jan 2020 21:45:11 +0000
Subject: [PATCH] opengl slide transitions not working with glm >= GLM 0.9.9.0

tracked it down to...

Removed default initialization, use GLM_FORCE_CTOR_INIT to restore the old behavior
so adding in GLM_FORCE_CTOR_INIT to get them working again

Change-Id: I1c6e7d8eb748fce40f0c518ff708708e5fb1e3d2
---
 slideshow/Library_OGLTrans.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/slideshow/Library_OGLTrans.mk b/slideshow/Library_OGLTrans.mk
index 4eca2a1ecaa3..9b64181d6a46 100644
--- a/slideshow/Library_OGLTrans.mk
+++ b/slideshow/Library_OGLTrans.mk
@@ -17,6 +17,7 @@ endif
 
 $(eval $(call gb_Library_add_defs,OGLTrans,\
     -DGLM_FORCE_RADIANS \
+    -DGLM_FORCE_CTOR_INIT \
 ))
 
 $(eval $(call gb_Library_use_sdk_api,OGLTrans))
-- 
2.24.1

